src/renderer/components/composer/DisabledMessageInput.tsx

```
@@ -7,40 +7,37 @@ type Props = {
   reason?: DisabledChatReasons
 }
 
-const DisabledMessageInput = forwardRef<any, Props>(
-  ({ reason }: Props, ref) => {
-    const tx = useTranslationFunction()
-
-    const reasonMessage = useMemo(() => {
-      switch (reason) {
-        case DisabledChatReasons.MAILING_LIST:
-          return tx('messaging_disabled_mailing_list')
-        case DisabledChatReasons.NOT_IN_GROUP:
-          return tx('messaging_disabled_not_in_group')
-        case DisabledChatReasons.DEADDROP:
-          return tx('messaging_disabled_deaddrop')
-        case DisabledChatReasons.UNKNOWN:
-        default:
-          return tx('messaging_disabled_unknown')
-      }
-    }, [reason, tx])
-
-    // If no reason was given we return no component
-    if (!reason) {
-      return null
+const DisabledMessageInput = ({ reason }: Props) => {
+  const tx = useTranslationFunction()
+
+  const reasonMessage = useMemo(() => {
+    switch (reason) {
+      case DisabledChatReasons.MAILING_LIST:
+        return tx('messaging_disabled_mailing_list')
+      case DisabledChatReasons.NOT_IN_GROUP:
+        return tx('messaging_disabled_not_in_group')
+      case DisabledChatReasons.DEADDROP:
+        return tx('messaging_disabled_deaddrop')
+      case DisabledChatReasons.UNKNOWN:
+        return 'UNKNOWN_DISABLED_CHAT_REASON'
```

We could also throw an Error here in case we arrive in the "unknown" state